stix_shifter_modules/trustar/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,70 @@
+{
+  "x-trustar-query": {
+    "fields": {
+      "x_searchTerm": ["searchTerm"],
```

Comment:
you don't need `x_` prefix for these properties name

Reply:
@mdazam1942
Implemented as suggested review it and merge it.